client/me/security-2fa-backup-codes/index.jsx

```diff
@@ -50,9 +50,7 @@ class Security2faBackupCodes extends React.Component {
 	onRequestComplete = ( error, data ) => {
 		if ( error ) {
 			this.setState( {
-				lastError: this.props.translate(
-					'Unable to obtain backup codes.  Please try again later.'
-				),
+				lastError: this.props.translate( 'Unable to obtain backup codes. Please try again later.' ),
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.